tests/test_hls_report.py

```diff
@@ -460,8 +460,8 @@ def test_spam_filter(vhls):
     _test_rpt(config)
 
 if __name__ == '__main__':
-    #test_knn_digitrec(False)
-    #test_kmeans(False)
+    test_knn_digitrec(False)
+    test_kmeans(False)
```

Comment:
The KMeans example is commented by yourself. Please uncomment the above `test_kmeans` function, or remove this line.

Reply:
The time I wrote the test script, I recall `kmeans` was unable to produce an HLS report. I'll either delete the test case or add back in after trying to make it work again.

Reply:
Okay, just comment L464.